perf-tests/clusterloader2/pkg/framework/config/client_config.go

```diff
@@ -35,16 +38,62 @@ const (
 	burst       = 200
 )
 
+var muxInitializedMultiClientConfig sync.Mutex
+
+var extraAPIServerAddresses []string
+
+func SetAPIServers(apiServerAddresses string) error {
+	muxInitializedMultiClientConfig.Lock()
+	defer muxInitializedMultiClientConfig.Unlock()
+	if len(extraAPIServerAddresses) > 0 {
+		return fmt.Errorf("Api Server addresses already set [%v]", extraAPIServerAddresses)
+	}
+
+	addrs := strings.Split(apiServerAddresses, ";")
+	for _, item := range addrs {
+		extraAPIServerAddresses = append(extraAPIServerAddresses, item)
+	}
+
+	if len(extraAPIServerAddresses) == 0 {
+		klog.Infof("Passed in api server addresses [%s]. Keep kubeconfig value instead.", apiServerAddresses)
```

Comment:
Rephrase this log please. The info is not clear to the reader. It would better to say "No api server addresses found in the passed-in string [%s] ...."

Reply:
I rephrased the word a bit.